linkerd/app/core/src/metrics/mod.rs

```diff
@@ -86,7 +87,7 @@ pub struct StackLabels {
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct RouteLabels {
     direction: Direction,
-    target: Addr,
+    addr: profiles::LogicalAddr,
```

Comment:
there's no need for the more general `Addr` type here -- we'll always have a `LogicalAddr` when we're dealing with routes.